services/github/src/handlers.js

```diff
@@ -489,17 +489,29 @@ async function statusHandler(message) {
     `Attempting to update status of the checkrun for ${organization}/${repository}@${sha} (${taskState.conclusion})`,
   );
   try {
+    const taskDefinition = await this.queueClient.task(taskId);
+    debug(`Result status. Got task build from DB and task definition for ${taskId} from Queue service`);
+    const {checkRunArtifactName} = taskDefinition.extra.github;
+
+    let checkRunArtifact = '';
+    if (checkRunArtifactName) {
+      checkRunArtifact = (await this.queueClient.getArtifact(taskId, runId, checkRunArtifactName)).toString();
+    }
+
+
     if (checkRun) {
       await instGithub.checks.update({
         ...taskState,
         owner: organization,
         repo: repository,
         check_run_id: checkRun.checkRunId,
+        output: {
+          title: `${this.context.cfg.app.statusContext} (${eventType.split('.')[0]})`,
+          summary: `${taskDefinition.metadata.description}`,
+          text: `[View task in Taskcluster](${taskUI(this.context.cfg.taskcluster.rootUrl, taskGroupId, taskId)})\n${checkRunArtifact}`,
```

Comment:
what does the check run artifact being on a newline here do? is it just the text name of the artifact?

Reply:
no, it's not the name, it's meant to be the contents of the artifact. (I might wanna add some logic to make sure it's a string)

Reply:
I know that `getArtifact` returns you the actual file. My initial thinking was it might be a good idea to avoid this and try [something like this](https://github.com/taskcluster/taskcluster/pull/2450/files#diff-41e7ecfbb590e5f3bbddd300fbf410dbR971) instead, but Dustin advised to use `getArtifact`, as the size of these artifacts is probably fine to hold in memory.

So, as I said in initial comment to this PR, I am expecting an artifact that contains a string, and I will get the string from there (probably takes more than `toString()` call, but that's something I was hoping to hear in the feedback)